gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -3220,7 +3220,8 @@ public void testIgnoreExceptionsOnCreateEmptyObject() throws IOException {
 
     gcs.createEmptyObject(
         RESOURCE_ID,
-        CreateObjectOptions.DEFAULT_OVERWRITE.toBuilder()
+        CreateObjectOptions.DEFAULT_OVERWRITE
```

Comment:
Is this `google-java-format` change?

Reply:
Idk, there is a maven build check for coding style and it is complaining about this